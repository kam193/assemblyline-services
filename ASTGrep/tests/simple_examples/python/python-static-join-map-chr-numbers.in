aPf3h=''.join(map(chr,[97,112,112,108,105,99,97,116,105,111,110,47,106,115,111,110]))
Hs3hf={''.join(map(chr,[67,111,110,116,101,110,116,45,116,121,112,101])):aPf3h}
