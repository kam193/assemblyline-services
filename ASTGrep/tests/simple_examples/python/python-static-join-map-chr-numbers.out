aPf3h='application/json'
Hs3hf={'Content-type':'application/json'}
